app/models/person.rb

```diff
@@ -191,7 +191,9 @@ class Person < ActiveRecord::Base # rubocop:disable Metrics/ClassLength
   ### Scopes
 
   scope :household, -> { where.not(household_key: nil) }
-
+  
+  scope :with_address, -> { where.not(address: nil, zip_code: nil, town: nil)
```

Comment:
was ist falls nur leerzeichen in einerm adressfeld vorhanden sind? dann sollten wir diese auch ausschliessen, oder?